packages/styles/scss/patterns/sections/card-section/_card-section.scss

```diff
@@ -43,8 +44,36 @@
   }
 }
 
+@mixin ratio-base() {
+  display: flex;
+
+  &::before {
+    content: '';
+    width: 1px;
+    margin-left: -1px;
+    float: left;
+    height: 0;
+    padding-top: aspect-ratio(4, 3);
```

Comment:
Small thing, but this probably shouldn't be apart of this mixin.

Reply:
@photodow  I did not understand this?

Reply:
Looking at what carbon is implementing this aspect is in a seperate mixin. Maybe we shouldn't focus on it, but at the least if you include it in this mixin you should allow the height and width there to be passed in at the mixin level above.